src/model/RestAction.ts

```diff
@@ -111,12 +116,14 @@ class RestAction implements Action {
                 :
                 null;
             let requestForm = this.form ? injectEvalAndVarsToMap(this.form, scenario.cache, ctx) : null;
+            let binaryData = this.dataBinary ? FS.createReadStream(this.dataBinary) : null
 
             request({
                 method:         this.method,
                 url:            injectEvalAndVarsToString(this.url, scenario.cache, ctx),
                 headers:        requestHeaders,
-                body:           requestBody,
+                body:           requestBody || binaryData,
```

Comment:
Does this mean that if any data is set, then any binaryData which is also set will silently be ignored? Maybe we should emit a warning in this case or at least document the behavior.

Reply:
I will document it in the Wiki